spark/src/main/java/org/apache/iceberg/spark/SparkDataFile.java

```diff
@@ -32,7 +33,7 @@
 import org.apache.spark.sql.Row;
 import org.apache.spark.sql.types.StructType;
 
-public class SparkDataFile implements DataFile {
+public class SparkDataFile implements DataFile, StructLike, Serializable {
```

Comment:
I think we won't have to make it serializable or implement `StructLike` if we use `SparkDataFile` as a wrapper.

Reply:
remove